recipe/meta.yaml

```diff
@@ -24,13 +24,12 @@ requirements:
     - setuptools >=64
     - setuptools-scm >=6.4
   run:
-    # - exceptiongroup  # >=3.6.2
+    - exceptiongroup
     - idna >=2.8
     - python >=3.7
     - sniffio >=1.1
     - typing_extensions
   run_constrained:
-    # - trio >=0.22  # >=3.6.2
```

Comment:
it does look like we should be pinning to `>=0.22`... should this come back in?

Reply:
I was initially confused by this, but looks like they pinned to `>=0.22` shortly _after_ the release; checking the 3.7.0 source, that constrains `<0.22`:

https://github.com/agronholm/anyio/blob/f7a880ffac4766efb39e6fb60fc28d944f5d2f65/pyproject.toml#L44

Reply:
oof, yep.